### DISCUSSION TITLE
Allow full width images in expandables

### DISCUSSION THREAD

**Discussion by liliakai** [GitbookIO/community#701] [0. General bugs and questions]
Seems like low hanging fruit since expandables already support inline images. Much desired by members of my organization!

**Comment by aikoii** [comment]
Hey @liliakai, thanks for sharing your feedback!

We have a similar suggestion here #16 , I encourage you to vote for it 🙂.

I must say, we don't have any immediate plans to support this yet.

(I will be closing this in favor of the first request)
